Makefile

```diff
@@ -4,6 +4,7 @@ NUGET_PACKAGE_NAME = nuget.71
 BUILD_CONFIGURATION = Debug
 BINARIES_PATH = $(shell pwd)/Binaries
 TOOLSET_TMP_PATH = $(BINARIES_PATH)/toolset
```

Comment:
`TMP` in the sense that it's deleted on every `git clean`, not that we manually clean it up each run.

I agree it's an overloaded term -- i'll choose another.